pages/api/pub-sub/publish.ts

```diff
@@ -0,0 +1,40 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
I've manually renamed the file, removed the unnecessary code for publish, and simplified the code via https://github.com/ably-labs/ably-nextjs-fundamentals-kit/commit/c9cc592f71930738e9bcf4d65bb7fe0b577475d8